code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -154,7 +156,60 @@
 
 	forceMove(Tsec)
 
+/**
+  * get_mangled_state() is relevant for flesh and bone bodyparts, and returns whether this bodypart has mangled skin, mangled bone, or both (or neither i guess)
+  *
+  * Dismemberment for flesh and bone requires the victim to have the skin on their bodypart destroyed (either a critical cut or piercing wound), and at least a hairline fracture
+  * (severe bone), at which point we can start rolling for dismembering. The attack must also deal at least 10 damage, and must be a brute attack of some kind (sorry for now, cakehat, maybe later)
+  *
+  * Returns: BODYPART_MANGLED_NONE if we're fine, BODYPART_MANGLED_SKIN if our skin is broken, BODYPART_MANGLED_BONE if our bone is broken, or BODYPART_MANGLED_BOTH if both are broken and we're up for dismembering
+  */
+/obj/item/bodypart/proc/get_mangled_state()
+	var/mangled_state = BODYPART_MANGLED_NONE
+	var/required_bone_severity = WOUND_SEVERITY_SEVERE
+
+	if(owner && owner.get_biological_state() == BIO_JUST_BONE && !HAS_TRAIT(owner, TRAIT_EASYDISMEMBER))
+		required_bone_severity = WOUND_SEVERITY_CRITICAL
+
+	// we can (generally) only have one wound per type, but remember there's multiple types
+	for(var/datum/wound/iter_wound in wounds)
+		if(istype(iter_wound, /datum/wound/blunt) && iter_wound.severity >= required_bone_severity)
+			mangled_state |= BODYPART_MANGLED_BONE
+		else if((istype(iter_wound, /datum/wound/slash) || istype(iter_wound, /datum/wound/pierce)) && iter_wound.severity >= WOUND_SEVERITY_CRITICAL)
```

Comment:
Remove the type check here, please.
Also, could you make this either a wound property (a flag or variable that determines its behavior) or a redefined proc?
Big istype checks don't really allow for very generic and expansible behavior.